data/nginx/vhosts/prod/app.conf.template

```diff
@@ -14,9 +14,14 @@ server {
 server {
 	gzip on;
 	gzip_min_length 1000;
-	gunzip on;
+	gunzip on; 
 	gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
+	listen 443 quic reuseport;
 	listen 443 ssl;
+	ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;
+	http2 on;
+	http3 on;
+	http3_hq on;
```

Comment:
Some browser optimizations, need to recompile a custom version of nginx to leverage caching.